router/helper.go

```diff
@@ -16,11 +16,13 @@ func GetPaginationSpec(r *http.Request) database.PaginationSpec {
 	}
```

Comment:
nit: setting `page = 1` on user mistake might not be advisable since having the user type `page=xyz` and showing results for that is undesirable

Reply:
Thanks for noticing the bug from previous merged PR, I agree we should put this on background to improve it on the next sprint!